release.yaml

```diff
@@ -19,8 +19,12 @@ rancher-logging:
 rancher-logging-crd:
   - 102.0.3+up3.17.10
 rancher-monitoring:
-  - 102.0.4-rc1+up40.1.2
+  - 102.0.4+up40.1.2
 rancher-monitoring-crd:
-  - 102.0.4-rc1+up40.1.2
+  - 102.0.4+up40.1.2
+rancher-cis-benchmark:
```

Comment:
Hmm. Actually maybe something off since validation failed.

Reply:
Actually realized this is just friday brain since the "thumbs up validation" is failing not the one that checks yaml.